fabric-keybindings-v0/src/main/java/net/fabricmc/fabric/api/client/keybinding/FabricKeyBinding.java

```diff
@@ -26,10 +30,40 @@
  * <p>
  * *ALL* instantiated FabricKeyBindings should be registered in
  * {@link KeyBindingRegistry#register(FabricKeyBinding)}!
+ * </p>
+ * <pre><code>
+ * FabricKeyBinding.Builder builder = new FabricKeyBinding.builder();
```

Comment:
Is it a good idea to encourage reuse of a builder type?

Reply:
Only did that for the example, but there is no reason why you shouldn't. The state of the builder is in no way linked to the keys you create with it, it simply holds the parameters, and there are benefits to being able to create a builder, use it to create a key, change one/two of its parameters, and create another key.

If you're _not_ planning to reuse the builder, it had might as well be an over-glorified constructor.